CHANGES.rst

```diff
@@ -36,6 +36,11 @@ associations
 
 - Remove ``MultilineLogger`` and no longer set it as the default logger. [#8781]
 
+background_sub
+--------------
+
+- Fixed crash when combining full and subarray observations for background subtraction. [#8787]
+
```

Comment:
```suggestion
```

now that #8671 is merged (switching change log handling to `towncrier`) this change log entry should be a file in `changes/` instead:
```shell
echo "Fixed crash when combining full and subarray observations for background subtraction." > changes/8787.background.rst
```

(new PRs will include instructions on how to do this)

Reply:
Sure, rebased and pushed